src/main/java/com/depromeet/buzz/participation/domain/Participation.java

```diff
@@ -0,0 +1,64 @@
+package com.depromeet.buzz.participation.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import com.depromeet.buzz.post.domain.Post;
+import com.depromeet.buzz.user.domain.User;
+
+@Entity
+public class Participation {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "user_id")
+	private User user;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "post_id")
+	private Post post;
+
+	public Post getPost() {
+		return post;
+	}
+
+	private Participation() {}
+
+	public Participation(User user, Post post) {
+		this.user = user;
+		this.post = post;
+	}
+
```

Comment:
`참여를 한다`라는 factory static method가  있으면 좋을 것 같아요!
생성자를 모두 private으로 만들구 

```suggestion
public static Participation participate(User user, Poast post) {
 return new Participation(user, post);
}
```

물론 없어도 되구요!!(bypass니까여)